python/ray/util/spark/cluster_init.py

```diff
@@ -1025,6 +1025,26 @@ def _setup_ray_cluster_internal(
     num_spark_task_gpus = int(
         spark.sparkContext.getConf().get("spark.task.resource.gpu.amount", "0")
     )
+    if num_spark_task_gpus > 0:
+        warn_msg = (
+            "You configured 'spark.task.resource.gpu.amount' to "
+            f"{num_spark_task_gpus},"
+            "we recommend setting this value to 0 by specifying this cluster "
+            "configuration when creating your cluster so that Spark jobs do not "
+            "reserve GPU resources, preventing Ray-on-Spark workloads to have the "
+            "maximum number of GPUs available."
```

Comment:
```suggestion
            "You configured 'spark.task.resource.gpu.amount' to "
            f"{num_spark_task_gpus},"
            "we recommend setting this value to 0 so that Spark jobs do not "
            "reserve GPU resources, preventing Ray-on-Spark workloads from having the "
            "maximum number of GPUs available."
```